src/test/kotlin/com/software/modsen/driverservice/controller/DriverControllerIntegrationTest.kt

```diff
@@ -0,0 +1,237 @@
+package com.software.modsen.driverservice.controller
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.software.modsen.driverservice.config.DatabaseContainerConfiguration
+import com.software.modsen.driverservice.config.KafkaContainerConfiguration
+import com.software.modsen.driverservice.dto.request.CarRequest
+import com.software.modsen.driverservice.dto.request.DriverRequest
+import com.software.modsen.driverservice.dto.response.CarResponse
+import com.software.modsen.driverservice.dto.response.DriverResponse
+import com.software.modsen.driverservice.dto.response.DriverWithCarResponse
+import com.software.modsen.driverservice.model.DriverSex
+import com.software.modsen.driverservice.util.ExceptionMessages
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.MethodOrderer
+import org.junit.jupiter.api.TestMethodOrder
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Import
+import org.springframework.http.MediaType
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.web.servlet.delete
+import org.springframework.test.web.servlet.get
+import org.springframework.test.web.servlet.put
+import org.springframework.test.web.servlet.post
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.web.servlet.function.RequestPredicates
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test")
+@AutoConfigureMockMvc
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+@Import(DatabaseContainerConfiguration::class, KafkaContainerConfiguration::class)
+class DriverControllerIntegrationTest {
+    @Autowired
+    lateinit var mockMvc: MockMvc
+
+    @Autowired
+    lateinit var objectMapper: ObjectMapper
+
+    @Test
+    fun `should return no content`() {
+        mockMvc.delete("/drivers/{id}", DEFAULT_ID)
+            .andDo { print() }
+            .andExpect {
+                status { isNoContent() }
+            }
+    }
+
+    @Test
+    @Order(1)
```

Comment:
Major: bad practice to use test ordering

Reply:
Rewrite tests without order